fastchat/model/model_adapter.py

```diff
@@ -504,10 +507,43 @@ def match(self, model_path: str):
 
     def get_default_conv_template(self, model_path: str) -> Conversation:
         return get_conv_template("h2ogpt")
+    
+class LoraAdapter(BaseAdapter):
+    def match(self, model_path: str):
+        # check if has adapter config in model_path
+        res = os.path.exists(os.path.join(model_path, "adapter_config.json"))
+        if res and 'lora' not in model_path.lower():
```

Comment:
random interjection.  If you're doing this, it'd be nice to be able to use the other PEFT adapter models as well.  Not all are LoRA, there's also prompt and prefix tuning along with llama style adapters.

It might be worth switching this to use "peft" as the key token for checking if it's a PEFT adapter rather than LoRA.